sections/git_commit_time.zsh

```diff
@@ -0,0 +1,48 @@
+#
+# Git commit_time
+#
+# Show time since previous commit
+# Inspired from "hyperzsh" theme
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_GIT_COMMIT_TIME_SHOW="${SPACESHIP_GIT_COMMIT_TIME_SHOW:=true}"
+SPACESHIP_GIT_COMMIT_TIME_PREFIX="${SPACESHIP_GIT_COMMIT_TIME_PREFIX:=since }"
+SPACESHIP_GIT_COMMIT_TIME_SUFFIX="${SPACESHIP_GIT_COMMITT_IME_SUFFIX:=}"
+SPACESHIP_GIT_COMMIT_TIME_THRESHOLD_LOW="${SPACESHIP_GIT_COMMIT_TIME_THRESHOLD_LOW:=30}" # in minutes
+SPACESHIP_GIT_COMMIT_TIME_THRESHOLD_HIGH="${SPACESHIP_GIT_COMMIT_TIME_THRESHOLD_HIGH:=240}" # in minutes
```

Comment:
I think sections use a different way to define default values in 3.0 now, without `:`, check the other sections